src/App.js

```diff
@@ -1,10 +1,10 @@
 import './App.css';
-import calculator from './components/calculator';
+import Calculator from './components/calculator';
 
 const App = () => (
   <div className="App">
     <header className="App-header">
-      {calculator()}
+      {Calculator()}
```

Comment:
[OPTIONAL] Hey 👋🏽  it is not recommended to call a React component directly like `{Calculator()}`. 🤔  Instead, you should render it as a JSX element like `<Calculator />`.

When you call a component like a regular function 🤙🏽 , you're not actually creating a `React component` instance, but instead, you're calling the function that returns `JSX` code. This can lead to unexpected behavior, such as losing state or re-rendering the entire component tree unnecessarily. 😱 

By rendering a component as a JSX element, you're creating a new instance of the component, which ensures that it will behave as expected and have access to its state and lifecycle methods. 👀